docs/docs/05-configuration.md

```diff
@@ -155,3 +162,48 @@ The short form of a full URL string is enough for general use. For advanced conf
 `on` is a special property for setting event handler functions on proxy server events. See the section on ["Listening for Proxy Events"](https://github.com/http-party/node-http-proxy#listening-for-proxy-events) for a list of all supported events. You must be using a `snowpack.config.js` JavaScript configuration file to set this.
 
 This configuration has no effect on the final build.
+
+#### Mount Options
+
+
+```js
+// snowpack.config.json
+{
+  "mount": {
+    // Files in the local src directory are written to `/_dist_/*` in the final build.
+    "src": "/_dist_",
+    // Files in the local public directory are written to `/*` in the final build.
+    "public": "/"
+  }
+}
+```
+
+The `mount` configuration lets you map local files to their location in the final build. If no mount configuration is given, then the entire current working directory (minus excluded files) will be built and mounted to the Root URL (Default: `/`, respects `baseUrl`).
+
+#### Alias Options
```

Comment:
💯 I know this went in a previous PR, but this seems to address our most hotly-requested feature.